environment/create_google_config.py

```diff
@@ -0,0 +1,308 @@
+		
+		# Continue with testing configuration
+		continue_test = raw_input('\nStep (6): Test your configuration\nDo you want to run a Hello WDL test to check your configuration? (yes or no) ')
+			
+		while not (continue_test.startswith("y") or continue_test.startswith("n")):
+			continue_test = raw_input('\nPlease answer yes or no: ').lower()
+
+		if continue_test.startswith("y"):
+			hello_test()
+
+		elif continue_test.startswith("n"):
+			sys.exit("Exiting.")
+			return
+
+	# Don't enable APIs, and exit
+	elif enable_apis.startswith("n"):
+		print "Don't forget to enable the APIs through the Google Console or gcloud SDK prior to using the configuration."
+		sys.exit("Exiting.")
+		return
+
+def enable_services(serviceName):
+	body = {"consumerId": "project:%s" % project_name}
+	params = {"serviceName": "%s" % serviceName, "body": body}
+	# print params
+	smgt.services().enable(**params).execute()
+
+def hello_test():
+	# Create WDL
+	print "Creating WDL file..."
+	wdl_ex = open("hello.wdl","w+")
+	#TODO: make tabs smaller
+	wdl_contents = "task hello {\n\tString addressee\n\tcommand {\n\t\techo \"Hello ${addressee}! Welcome to Cromwell . . . on Google Cloud!\"\n\t}\n\toutput {\n\t\tString message = read_string(stdout())\n\t}\n\truntime {\n\t\tdocker: \"ubuntu:latest\"\n\t}\n}\n\nworkflow wf_hello {\n\tcall hello\n\n\toutput {\n\t\thello.message\n\t}\n}"
+	wdl_ex.write(wdl_contents)
+	wdl_ex.close()
+	print "Your WDL file is ready! It is stored as hello.wdl."
+
+	# Create Inputs file
+	print "Creating inputs file..."
+	inputs_to_wdl = open("hello.inputs", "w+")
+	#TODO: make tabs smaller
+	#TODO: check \ necessary?
+	inputs_contents = "{\n\t\"wf_hello.hello.addressee\": \"World\"\n}"
+	inputs_to_wdl.write(inputs_contents)
+	inputs_to_wdl.close()
+	print "Your inputs file is ready! It is stored as hello.inputs."
+
+	# Download latest Cromwell
+	print "Downloading latest version of Cromwell execution engine..."
+	r = requests.get('https://api.github.com/repos/broadinstitute/cromwell/releases/latest')
+	s = r.json()
+	t = json.dumps(s)
+	for asset in s["assets"]:
+		if "cromwell-" in asset["browser_download_url"]:
+			download_url = asset["browser_download_url"]
+			urllib.urlretrieve(download_url, "cromwell.jar")
+
+	#TODO: add error handling for if cromwell doesn't download
+
+	# Run test
+	home = os.path.expanduser("~")
+	test_configuration = "java -Dconfig.file=" + home +"/.google_cromwell.config -jar cromwell.jar run hello.wdl -i hello.inputs"
+	print "Cromwell is downloaded and ready for operation.\n\nStarting Hello World test.\n\nRunning $ %s" % test_configuration
+	os.system(test_configuration)
+
+	# Success
+	print "Workflow succeeded!\nOutputs for this workflow can be found in gs://%s\n\nYou have successfully set up your Google Project, Bucket, and configuration. \nCheck out the WDL website for more information on writing your own workflows: https://software.broadinstitute.org/wdl/documentation/quickstart.\n Happy WDL-ing!" % bucket_name
```

Comment:
we should check here that it actually succeeded or not.  if it did not we should give info about what to do now